README.md

```diff
@@ -1,69 +1,75 @@
-- Couchbase .NET SDK 2.3.10 or greater
-- Common.Logging 3.3.1 or greater
-- Common.Logging.Core 3.3.1 or greater
-- Newtonsoft JSON 9.0.1 or greater
-- re-linq 2.1.1
+- Couchbase .NET SDK 3.0.3 or greater
 
-If you are using NuGet, then the dependencies (other than Couchbase Server) will be installed for you via the package manager.
+### Installing Couchbase Server
 
-### Installing Couchbase Server ###
 For a single instance of Couchbase Server running on localhost, you can [download Couchbase Server here](https://www.couchbase.com/downloads) (make sure it's 4.0 or later). If you would like to create a cluster, the easiest way is by using the [Vagrant scripts for provisioning clusters](https://github.com/couchbaselabs/vagrants). Additionally, you can use [Docker scripts](https://hub.docker.com/r/couchbase/server/). Follow the directions on each respective link for installation information.
 
-### Installing the package using NuGet ###
-Once you have a Couchbase Server instance or cluster setup, open Visual Studio 13 or greater or [MonoDevelop](http://www.monodevelop.com/) and create a .NET or .NET Core application. Open the NuGet Package Manager and search for "Couchbase Linq" or type the following into the Package Manager console:
+### Installing the package using NuGet
+
+Once you have a Couchbase Server instance or cluster setup, open Visual Studio 2019 or greater or [MonoDevelop](http://www.monodevelop.com/) and create a .NET or .NET Core application. Open the NuGet Package Manager and search for "Couchbase Linq" or type the following into the Package Manager console:
 
     Install-Package Linq2Couchbase
 
-NuGet will install the package and all dependencies. Once you have the resolved the dependencies, you will initialize a `ClusterHelper` object which will manage the bucket resources needed by the Linq provider.
+## Quick Start
 
-## Quick Start ##
 This Quick Start assumes that you have [already installed the travel-sample bucket](https://docs.couchbase.com/server/current/manage/manage-settings/install-sample-buckets.html), which is available and built-in to Couchbase Server.
 
 Query the 'travel-sample' bucket and return 10 airlines in any order:
 
-    ClusterHelper.Initialize(new ClientConfiguration
-    {
-         Servers = new List<Uri> {new Uri("http://localhost:8091/")}
-    }, new PasswordAuthenticator("Administrator", "password"));
+```cs
+static async void Main() {
+    var cluster = await Cluster.ConnectAsync("couchbase://localhost", options => {
+        options.UserName = "Administrator";
+        options.Password = "password";
 
-    var context = new BucketContext(ClusterHelper.GetBucket("travel-sample"));
+        options.AddLinq();
+    });
+
+    // Wait until the cluster is bootstrapped
+    await cluster.WaitUntilReadyAsync(TimeSpan.FromSeconds(10));
+
+    var context = new BucketContext(await cluster.BucketAsync("travel-sample"));
```

Comment:
Should Cluster have an extension method that creates BucketContexts? 

`cluster.GetBucketContext(string name)` or `cluster.GetQueryContext(string bucketName)`?

Open to counter-ideas here.

Reply:
That could be interesting. I'd probably go with CreateBucketContext to make clear it's making a new one. I'd also think an extension on IBucketProvider from the DI extensions would be good, too. Feels like a separate PR, though.

Reply:
@brantburnett - this ready for review?

Reply:
I think so, up to you if you want me to add more to it now or in a separate PR.